src/frontend/package.json

```diff
@@ -24,6 +24,7 @@
     "@fortawesome/free-solid-svg-icons": "6.2.0",
     "@fortawesome/react-fontawesome": "0.2.0",
     "breakpoint-sass": "3.0.0",
+    "esbuild-darwin-arm64": "0.15.11",
```

Comment:
This was added automatically because I use macOS and is for bundling TypeScript.